<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Complex</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/baha.png</normaloff>../../../Downloads/baha.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Complex number 1 :</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Real</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_real_1</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_real_1"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Imaginary</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_imaginary_1</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_imaginary_1"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Complex number 2 :</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Real</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_real_2</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_real_2"/>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Imaginary</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_imaginary_2</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_imaginary_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ok">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Choose your operation :</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_sum">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_sub">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Substraction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_div">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Division</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_mul">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Multiplication</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QTextBrowser" name="result"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear_result">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
